<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrakeSelfCheckPage</class>
 <widget class="QWidget" name="BrakeSelfCheckPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;
}
QPushButton:disabled
{
		border-radius:5px;
	font: 20px, &quot;SimHei&quot;;
    color: grey;
	background-color: rgb(221,221,221);
}
	</string>
  </property>
  <widget class="QLabel" name="label_Bogie3_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bogie2_1">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>230</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR2_DI1">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>140</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slideDetect">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:black;</string>
   </property>
   <property name="text">
    <string>自检内容</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR4_DI1">
   <property name="geometry">
    <rect>
     <x>757</x>
     <y>140</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bogie1_1">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>230</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_No">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>140</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>车号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_BogieCut">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>230</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>转向架自检结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR3_DI1">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>199</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bogie4_2">
   <property name="geometry">
    <rect>
     <x>757</x>
     <y>230</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_CAR1_DI1">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>140</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:black;</string>
   </property>
   <property name="text">
    <string>按下［开始］键，进行制动自检。按下［停止］键，停止制动自检。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:black;</string>
   </property>
   <property name="text">
    <string>［注意］自检进行中请保持本页面，切换到其他页面会终止自检。</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>300</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cease">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>550</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bogie1_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>230</y>
     <width>86</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bogie2_2">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>230</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bogie3_1">
   <property name="geometry">
    <rect>
     <x>658</x>
     <y>230</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bogie4_1">
   <property name="geometry">
    <rect>
     <x>846</x>
     <y>230</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_antiSlip">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>防滑自检</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_brakeDetect">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>制动自检</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_serialDetect">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>连续自检</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_errorInfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:black;
color: red;
font:20px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>自检条件不满足，请检查以下条件是否满足：
1 列车未施加紧急制动   2 列车未施加强迫缓解（强迫缓解按钮灯灭）
3 列车处于静止状态     4 总风压力大于7.0bar
5 所有车辆制动储风缸压力大于6.5bar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>170</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>未自检时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>377</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_time1">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>170</y>
     <width>369</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_prepare">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>200</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>自检准备</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prepare2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>377</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_prepare1">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>200</y>
     <width>369</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_slideDetect_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:black;</string>
   </property>
   <property name="text">
    <string>自检准备：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slideDetect_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_slideDetect_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_slideDetect_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:black;</string>
   </property>
   <property name="text">
    <string>条件满足</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slideDetect_6">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>450</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:black;</string>
   </property>
   <property name="text">
    <string>条件未满足</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_timeout">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>630</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:black;
color: red;
font:20px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>自检超时！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_keepBrake">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>420</y>
     <width>145</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>保持制动切除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
